_positions/TAM-cloud.gov.md

```diff
@@ -0,0 +1,43 @@
+
+---
+title: Technical Account Manager — Cloud.gov 
+permalink:
+team: Infrastructure
+active: false
```

Comment:
Needs a permalink and to have `active` set to `true` — @hestherchang, mind if I hop in and make those changes?